server_admin/topics/authentication/flows.adoc

```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
 one of the specified levels. If it is not able to return one of the specified levels (For example if the requested level is unknown or bigger than configured conditions
 in the authentication flow), then {project_name} will throw an error.
 
+[[_user_session_limits]]
+*User session limits*
+
+Limits on the number of session that a user can have can be configured. Sessions can be limited per realm or per client.
+
+To add session limits to a flow perform the following steps.
+
+. Click *Add execution* for the flow.
+. Select *User Session Count Limiter* from the item list.
+. Click *Save*.
+. Click *Required* for the *User Session Count Limiter* authentication type to set its requirement to required.
+. Click *Actions* for the *User Session Count Limiter*.
+. Click *Config*.
+. Enter an alias for this config.
+. Enter the required maximum number of sessions a user can have in this realm. If a value of 0 is used then this check is disabled.
```

Comment:
should be 
If a value of 0 is used, this check is disabled.

" then" is not needed. Also, a comma should conclude an introductory phrase.